tests/Unit/ParallelAlgorithms/Initialization/Actions/Test_AddSimpleTags.cpp

```diff
@@ -0,0 +1,103 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <string>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "Framework/ActionTesting.hpp"
+#include "Parallel/Actions/SetupDataBox.hpp"
+#include "Parallel/GlobalCache.hpp"
+#include "Parallel/PhaseDependentActionList.hpp"
+#include "ParallelAlgorithms/Initialization/Actions/AddSimpleTags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace {
+struct SomeNumber : db::SimpleTag {
+  using type = double;
+};
+
+struct SomeOtherNumber : db::SimpleTag {
+  using type = double;
+};
+
+struct SquareNumber : db::SimpleTag {
+  using type = double;
+};
+
+struct AddSomeAndOtherNumber {
+  using return_tags = tmpl::list<SomeNumber, SomeOtherNumber>;
+  using argument_tags = tmpl::list<>;
+
+  static void apply(const gsl::not_null<double*> some_number,
+                    const gsl::not_null<double*> some_other_number) {
+    *some_number = 2.;
+    *some_other_number = 3.;
+  }
+};
+
+struct AddSquareNumber {
+  using return_tags = tmpl::list<SquareNumber>;
+  using argument_tags = tmpl::list<SomeNumber>;
+  static void apply(const gsl::not_null<double*> square_number,
+                    const double some_number) {
+    *square_number = some_number * some_number;
+  }
+};
+
+template <typename Metavariables>
+struct Component {
+  using metavariables = Metavariables;
+  using chare_type = ActionTesting::MockArrayChare;
+  using array_index = int;
+
+  using phase_dependent_action_list = tmpl::list<
+      // Parallel::PhaseActions<
```

Comment:
Can this commented `PhaseActions` be removed?

Reply:
Yes, thanks for catching this! I've squashed in its removal :)